charts/cadc-tap/templates/uws-db-deployment.yaml

```
@@ -1,3 +1,4 @@
+{{ if .Values.uws.enabled -}}
```

These could then be
```suggestion
{{ if (not .Values.cloudsql.enabled) -}}
```
just so that there's one less configuration setting to keep in sync.